doc/torque_api.md

```diff
@@ -0,0 +1,174 @@
+
+TorqueLayer currently expects `marker` styling.
+
+#### Example
+
+This is how a minimal example of a stylesheet for a Torque visualisation would look like.
+
+```css
+Map {
+  -torque-time-attribute: "date";
+  -torque-aggregation-function: "count(cartodb_id)";
+  -torque-frame-count: 760;
+  -torque-animation-duration: 15;
+  -torque-resolution: 2;
+}
+#layer {
+  marker-width: 3;
+  marker-fill-opacity: 0.8;
+  marker-fill: #FEE391; 
+}
+#layer[value = 4] { // Use of the value variable, generated by the function specified in -torque-aggregation-function
+  marker-fill: #FABADA; 
+}
+#layer[zoom = 12] { // Use of the zoom variable
+  marker-width: 10;
+}
+```
+
+### Data Methods
+
+Method | Options | Returns | Description
+---|---|---|---
+`setSQL(sql statement)` | `SQL string` | `this` | Change the SQL on the data table (not available with named maps)
+`error(callback)` | `callback function with a list of errors as argument` | `this` | specifies a callback function to run if there are query errors
+
+#### Example
+
+SQL Example to limit the data used in the Torque map.§
+
+```js
+torqueLayer.setSQL("SELECT * FROM table LIMIT 100");
+```
+
+### Events
+
+_**Note:** You can only run events after the [required libraries](/cartodb-platform/torque/torquejs-getting-started/#advanced-torquejs-libraries) are loaded. Otherwise, the [interaction methods](/cartodb-platform/torque/torque-interaction-methods/) will not work._
+
+Events in Torque follow the format:
+
+```js
+torqueLayer.on('event-type', function([callback_obj]) {
+  // do something
+});
+```
+
+Events | Callback Object | Description
+---|---|---
+`change:steps` | current step | When a map changes steps, this event is triggered
+`change:time` | current time, step number | When a map changes time, this event is triggered
+`play` | none | Triggered when the Torque layer is played
+`pause` | none | Triggered when the Torque layer is paused
+`stop` | none | Triggered when the Torque layer is stopped
+`load` | none | Triggered when the Torque layer is loaded
+
+#### Example
+
+An event example to print the current step to the console log.
+
+```js
+torqueLayer.on('change:steps', function(step) {
+  // do something with step
+  console.log('Current step is ' + step);
+});
+```
+
+## Google Maps Layers
+
+### GMapsTorqueLayer(_options_)
+
+This class does exactly the same as ``L.TorqueLayer`` but using Google Maps instead. The main difference is that this class
+is not a layer but is an overlay, so in order to add it to the a map use, ``layer.setMap`` instead of ``overlayMapTypes``. See the [Overlay View](https://developers.google.com/maps/documentation/javascript/reference#OverlayView) reference in Google Maps API doc.
+
+#### Options
+
+Name | Description
+--- | ---
+map | A google.maps.Map instance
+
+
+## Torque Functions
```

Comment:
I'm removing this section for two reasons:
1. It's linking to a particular line within the CartoDB.js codebase. Most of the methods in that context are private and not for public use.
2. They're CartoDB.js methods. Even if they are related to Torque, explaining them is outside the scope of the Torque.js documentation.